packages/grid/x-data-grid-pro/src/colDef/gridRowReorderColDef.tsx

```diff
@@ -0,0 +1,20 @@
+import { GridColDef, GRID_STRING_COL_DEF } from '@mui/x-data-grid';
+import { renderRowReorderCell } from '../components/GridRowReorderCell';
+
+export const GRID_REORDER_COL_DEF: GridColDef = {
+  ...GRID_STRING_COL_DEF,
+  field: '__reorder__',
+  type: 'reorder',
+  sortable: false,
+  filterable: false,
+  width: 50,
+  align: 'center',
+  headerAlign: 'center',
+  headerName: ' ',
```

Comment:
We need a name.

<img width="300" src="https://user-images.githubusercontent.com/42154031/164351322-607fa5b9-ca83-47bf-a99d-07a758a4e887.png" alt="image" style="max-width: 100%;">